modules/apps/commerce/commerce-frontend-js/test/dev/components/DatasetDisplay.js

```diff
@@ -546,7 +553,7 @@ const productsDataSetDisplayProps = {
 		{
 			id: 'blbl',
 			label: 'Custom Filter',
-			moduleUrl: '/blblasd/asd/basdkj',
+			moduleURL: '/blblasd/asd/basdkj',
```

Comment:
This also looks like a "test" value, so I would also either set it as an empty string or an actual URL.